ui/src/message_popup/mobile_prototype_card.jsx

```diff
@@ -0,0 +1,239 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+import _ from 'lodash';
+
+import {withStudents} from './transformations.jsx';
+
+//Material-UI imports
+import AppBar from 'material-ui/AppBar';
+import Paper from 'material-ui/Paper';
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+import IconButton from 'material-ui/IconButton';
+import Divider from 'material-ui/Divider';
+import TextField from 'material-ui/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+
+//Icons
+import FaceIcon from 'material-ui/svg-icons/action/face';
+import InfoOutlineIcon from 'material-ui/svg-icons/action/info-outline';
+
+
+export default React.createClass({
+  displayName: 'MobilePrototypeCard',
+  
+  propTypes: {
+    question: React.PropTypes.object.isRequired
+  },
+  
+  getInitialState(){
+    return ({
+      question: this.props.question,
+      currentStudent: undefined,
```

Comment:
This is more about the dialog than the current student, so maybe `dialogForStudent`?